pkg/runner/exec.go

```diff
@@ -88,13 +89,17 @@ func runBenchmark(cfg config.Config, clusterMetadata ocpmetadata.ClusterMetadata
 		}
 		wg.Wait()
 		genResultSummary(&result)
-		log.Infof("Summary for %s: Rps=%.0f avgLatency=%.0f μs P99Latency=%.0f μs", cfg.Termination, result.TotalAvgRps, result.AvgLatency, result.P99Latency)
+		aggAvgRps += result.TotalAvgRps
+		aggAvgLatency += result.AvgLatency
+		aggP99Latency += result.P99Latency
+		log.Infof("Rps=%.0f avgLatency=%.0f μs P99Latency=%.0f μs", result.TotalAvgRps, result.AvgLatency, result.P99Latency)
```

Comment:
Add back the termination type. Otherwise in a stream of output it is difficult to see what's being tested.